### PR TITLE
filled out readme and tests

### PR DESCRIPTION
**Pull request by anitarua**
Let me know if there's anything else that should be added to the README

### CODE REVIEW

File: test/integration.test.ts [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+    if (!endpoint) {
+      throw new Error('Missing environment variable MOMENTO_HTTP_ENDPOINT');
+    }
+
+    const momentoClient = new HttpClient(authToken, endpoint);
+    const key = 'foo';
+    const value = 'bar';
+
+    const setResponse = await momentoClient.set(cacheName, key, value);
+    expect(setResponse).toEqual(204);
```
Comment (bruuuuuuuce) [inline test/integration.test.ts:25]: I'm just wondering, why are the responses `204`? I would expect it to be a `200` or `201`

Comment (bruuuuuuuce) [reply test/integration.test.ts:25]: 204 is usually what is returned from CORs

Comment (anitarua) [reply test/integration.test.ts:25]: I was following the [docs](https://docs.momentohq.com/develop/api-reference/http-api#set-value) and it said set and delete would return 204 when successful

Comment (bruuuuuuuce) [reply test/integration.test.ts:25]: giphy

**Review by bruuuuuuuce — APPROVED** [review]
:shipit:

**Pull request merged**